pkg/fortify/fortify.go

```diff
@@ -76,6 +76,7 @@ type SystemInstance struct {
 
 // NewSystemInstance - creates an returns a new SystemInstance
 func NewSystemInstance(serverURL, apiEndpoint, authToken string, timeout time.Duration) *SystemInstance {
+	serverURL = strings.TrimRight(serverURL, "/")
```

Comment:
> To remove a suffix, use TrimSuffix instead.

https://golang.org/pkg/strings/#TrimRight

is this the right function?

Reply:
No, I do mean `TrimRight()`. We want to make sure that the URL does not have a trailing slash. No matter if the user configures `...ssc//` or `...ssc/`. `TrimRight()` and `TrimSuffix()` work very differently. `TrimRight()` takes a "cutset" whereas `TrimSuffix()` takes a literal. TrimRight() removes every rune from the end of a string, until it finds one that is not in the "cutset".

Reply:
Ok, so practically speaking, the difference is that Trimright can handle multiple slashes in the end, whereas TrimSuffix with argument `/` would only trim a single one. Seems correct to me.

Reply:
Yes.